Example/Backpack/View Controllers/RatingsViewController.swift

```diff
@@ -25,18 +25,48 @@ class RatingsViewController: UIViewController {
     var layout: BPKRatingLayout = .horizontal
 
     fileprivate static var ratingData = [
-        (ratingValue: 3.0, title: "Low title", subtitle: "Low subtitle",
+        (ratingValue: 3.0, titleDefinition: RatingTextDefinition(highRatingText: "",
```

Comment:
Instead of these can we just define some variables `titleDefinition` and `subtitle definition` and use them for each of these?